public/locales/en-US/grafana.json

```diff
@@ -1,5 +1,4 @@
 {
-  "_comment": "The code is the source of truth for English phrases. They should be updated in the components directly, and additional plurals specified in this file.",
```

Comment:
should this be removed? the comment that generates it has been moved to the `@grafana/i18n` package, maybe it needs to stay in core for this comment to persist? 🤔

Reply:
This was removed by the tooling, `make extract-i18n`  🤷

Reply:
right, but i think that's because there's a comment that's [now on line 42 of `src/i18n.tsx`](https://github.com/grafana/grafana/pull/103858/files#diff-1ad4ced047a546633fbb40142658c6af6bf2e73a6497a04ec3c2e3a130bf9d52R42) that was generating it. maybe we don't want to move that comment? 🤔

Reply:
Good catch,  I'll add it back 👍